tests/scaletest/scale_test.go

```diff
@@ -199,15 +205,14 @@ func RebootAko(t *testing.T, wg *sync.WaitGroup) {
 	t.Logf("Rebooting AKO pod %s of namespace %s ...", akoPodName, AKONAMESPACE)
 	err := lib.DeletePod(akoPodName, AKONAMESPACE)
 	if err != nil {
-		t.Fatalf("Cannot reboot Ako pod as : %v", err)
-		os.Exit(0)
+		ExitWithErrorf(t, "Cannot reboot Ako pod as : %v", err)
 	}
 	t.Logf("Ako rebooted.")
 	defer wg.Done()
 }
 
 /* Reboots Controller/Node/Ako if Reboot is set to true */
-func CheckReboot(t *testing.T, wg *sync.WaitGroup) {
+func (t *testing.T, wg *sync.WaitGroup) {
```

Comment:
We are missing function name here.

Reply:
thanks, i'll check this out, it's still in progress though, will add reviewers later

Reply:
Fixed